app/views/roles/_organisations.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if role.organisations_present? %>
+    <div class="organisations-list">
+      <div class="organisations-list">
```

Comment:
It looks like these are the same divs. Do we use the `organisations-list` class anywhere as well?

Reply:
Oh you're right! No we don't use it anywhere else. There are others similar to it for the Organisation page but not the same styling. Do you think it would be better to call it `roles-organisations-list`?

Reply:
I was thinking if we don't use it anywhere we could just leave it as a `<div>` without a class.

Reply:
Ok 👍